uk_bin_collection/uk_bin_collection/councils/ForestOfDeanDistrictCouncil.py

```
@@ -1,132 +1,142 @@
+                        )
+                        raw_date = (
+                            td.get("data-cell-value", "").strip()
+                            or td.get_text(strip=True)
+                        )
+                        if container_type and raw_date:
+                            data["bins"].append({
+                                "type": container_type,
+                                "collectionDate": self._parse_date(raw_date, current_year),
+                            })
+                    except Exception:
+                        continue
```

Ruff (0.15.12)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)